### ISSUE TITLE
Change the plugin name?

### ISSUE THREAD

**Issue by proItheus** [artisticat1/obsidian-latex-suite#59]
I think current name of the plugin is a bit confusing about what it actually **can** do.
It is already a full-featured snippet engine for obsidian, completely overpowers `textsnippets`. In fact, it will be almost just as powerful as `UltiSnips` after the script support #16 is implemented. It maybe better to have a more kind of "universal" name, or at least add a word `snippet` to the name for people searching for a snippet engine to find it.

I have had a hard time struggling to find something like `ultisnips` in obsidian. Finally, I gave up and only expected to find something to speed up my latex writing, and then I found this plugin. Surprise! There's already a powerful snippet engine for obsidian, just that it doesn't have any word related to `snippet` in its name:joy:

Finally,thanks for your wonderful plugin! This has make my life with obsidian **a lot**  easier!!

**Comment by artisticat1** [comment]
I agree! The plugin's name doesn't fully represent what it can do. I did actually consider naming the plugin something more universal (like you suggest), but it comes with so many additional latex-related features that I went with this.

I could've gone with a more universal name and marketed the plugin as a general snippet engine, but I would've had to add some sort of note that explained to users who aren't interested in latex that they could turn off/ignore all the latex-related features. I am not sure how clean a solution this is.

Adding "snippets" somewhere in the plugin's description might be a good solution for now. Any suggestions are welcome.

Glad you are enjoying the plugin! :)
